docs/components/popover.md

```diff
@@ -300,6 +346,22 @@ triggers)
   export default {
     data () {
       return {
+        selectedPosition: 'auto',
```

Comment:
these are added to this block because vuepress isn't picking up the data if its added somewhere else in the doc